### PR TITLE
[Admin] Define GpsSystemTests

### PR DESCRIPTION
**Pull request by brendanfraser597**
Working on #55 .

### CODE REVIEW

File: docs/uml-class-diagrams/admin/test/GpsSystemTests/GpsSystemTests.puml [hunk 1]
```diff
@@ -2,7 +2,21 @@
 
 package edu.oakland.test.admin {
     class GpsSystemTests {
-
+        + void initDisplayGPSInterfaceNotNull()
+        + void initSatelliteNamesNotNull()
```
Comment (andrewdimmer) [inline docs/uml-class-diagrams/admin/test/GpsSystemTests/GpsSystemTests.puml:6]: Let's validate the empty edge case as well.
```suggestion
        + void initSatelliteNamesNotNull()
        + void initSatelliteNamesNotEmpty()
```

File: docs/uml-class-diagrams/admin/test/GpsSystemTests/GpsSystemTests.puml [hunk 2]
```diff
@@ -2,7 +2,21 @@
...
+        + void gpsSystemsConstructedSuccessfully()
+        + void useCase1RunSuccessfully()
+        + void uC1ScannerNotNull()
+        + void useCase2RunSuccessfully()
+        + void uC2ScannerNotNull()
+        + void satellietNamesNotNull()
```
Comment (andrewdimmer) [inline docs/uml-class-diagrams/admin/test/GpsSystemTests/GpsSystemTests.puml]: We can remove that if we're removing this as a method parameter in favor of just having that in the constructor.
```suggestion
```

File: docs/uml-class-diagrams/admin/test/GpsSystemTests/GpsSystemTests.puml [hunk 3]
```diff
@@ -2,7 +2,21 @@
...
+        + void incrementalLatAmountNotNull()
+        + void incrementLngAmountNotNull()
+        + void LocationDataPointNotNull()
```
Comment (andrewdimmer) [inline docs/uml-class-diagrams/admin/test/GpsSystemTests/GpsSystemTests.puml]: Just a quick naming clarification.
```suggestion
        + void initLocationDataPointNotNull()
```

File: docs/uml-class-diagrams/admin/test/GpsSystemTests/GpsSystemTests.puml [hunk 4]
```diff
@@ -2,7 +2,21 @@
...
 package edu.oakland.test.admin {
     class GpsSystemTests {
-
+        + void initDisplayGPSInterfaceNotNull()
+        + void initSatelliteNamesNotNull()
+        + void gpsSystemsConstructedSuccessfully()
+        + void useCase1RunSuccessfully()
+        + void uC1ScannerNotNull()
+        + void useCase2RunSuccessfully()
+        + void uC2ScannerNotNull()
```
Comment (andrewdimmer) [inline docs/uml-class-diagrams/admin/test/GpsSystemTests/GpsSystemTests.puml:12]: We may need more test cases here for what happens for each connection, but that may play out in integration testing instead... this is good enough for now, just be aware if we make this method more than just a single path, we'll need multiple test cases.

**Review by andrewdimmer — CHANGES_REQUESTED** [review]
Couple changes, mostly just naming. Otherwise, looks good!

**Review by andrewdimmer — APPROVED** [review]
Looks good to me, you're all set to merge!

**Review by TessaPeruzzi — APPROVED** [review]
